source/payment_flow/index.html.md.erb

```diff
@@ -318,30 +318,23 @@ The `state` array within the JSON lets you know the outcome of the payment:
 See the [__Making payments__](/making_payments/#making-payments) section for
 more information about how you can integrate your service with GOV.UK Pay.
 
-## Resume a payment
+### Resuming an incomplete payment
 
-If your user starts a payment but does not complete it, they can resume that
-incomplete payment. An example of this could be if they use an internet
-browser’s ‘back’ button during a payment, and then go forward by selecting
-links on your website.
+You can send your user back to GOV.UK Pay if they arrive at your `return_url` before they've finished their payment. For example, your user selected their web browser’s **Back** button while making a payment, then selected **Forward**.
 
-If your service uses the resume payment feature, you will:
+You can only resume a payment if the `status` is one of the following:
 
-* minimise the number of expired payments
+- `created`
+- `started`
+- `submitted`
 
-* reduce the number of unnecessary new payments
+Resume the payment by [calling the GOV.UK API to find the payment](/reporting/#get-information-about-a-single-payment), then redirecting your user to the new `next_url` from the API response.
 
-When your user resumes a payment, the `next_url` will take them to a screen that
-is appropriate for their payment’s current status. For example, a payment with
-a `started` status will resume at the card details input page. The `next_url`
-is a one-time URL. If a payment has already resumed using a `next_URL`, that
-URL will not be usable again.
+You should not reuse the `next_url` you received when you first created the payment. Each `next_url` has a unique token so you can only use it once.
 
-A payment cannot resume if it has a status of `success`, `failed`, `cancelled`
-or `error`. If your service tries to resume a payment of this type, your user
-will be sent to your service’s `return_url`. The `return_url` is the URL of a
-page on your service that GOV.UK Pay sends your user to after they have
-completed their payment attempt.
+We'll take your user to a screen that's appropriate for their payment’s current status. For example, if the `status` of the payment is `started`, we'll show the **Enter card details** page.
```

Comment:
The example might be more meaningful if the page the user gets sent to isn’t the first one in the flow. Maybe use `submitted` and the confirm page?

Reply:
👍 Updated in https://github.com/alphagov/pay-tech-docs/pull/332/commits/7924b40b4e4d63cc7d3dd4078cc21be51d6e28b6